eng/Versions.props

```
@@ -250,9 +250,9 @@
     <PlaywrightSharpVersion>0.192.0</PlaywrightSharpVersion>
     <PollyExtensionsHttpVersion>3.0.0</PollyExtensionsHttpVersion>
     <PollyVersion>7.2.2</PollyVersion>
-    <SeleniumSupportVersion>4.0.0</SeleniumSupportVersion>
-    <SeleniumWebDriverChromeDriverVersion>95.0.4638.1700</SeleniumWebDriverChromeDriverVersion>
-    <SeleniumWebDriverVersion>4.0.0</SeleniumWebDriverVersion>
+    <SeleniumSupportPackageVersion>4.1.0</SeleniumSupportPackageVersion>
+    <SeleniumWebDriverChromeDriverPackageVersion>97.0.4692.7100</SeleniumWebDriverChromeDriverPackageVersion>
+    <SeleniumWebDriverPackageVersion>4.1.0</SeleniumWebDriverPackageVersion>
```

Do we need to update usage as well given property name change.

---

Property name change was a mistake&hellip;
```suggestion
    <SeleniumSupportVersion>4.1.0</SeleniumSupportVersion>
    <SeleniumWebDriverChromeDriverVersion>97.0.4692.7100</SeleniumWebDriverChromeDriverVersion>
    <SeleniumWebDriverVersion>4.1.0</SeleniumWebDriverVersion>
```